public/js/user.js

```diff
@@ -2,7 +2,7 @@
 function updateUser(id) {
   const floorEl = document.getElementById('floor-input')
   const floor = parseInt(floorEl.value)
-  if (!isNaN(floor) && (floor < 3 || floor > 18)) {
+  if (isNaN(floor) || floor < 3 || floor > 18) {
     displayMessage('danger', 'A szint üres vagy 3 és 18 közötti szám lehet')
   } else {
```

Comment:
Here, we declare, the floor can be blank, but we do not actually allow it in this logic. (It's good on server side.) I suggest this modification:
```javascript
const floor = floorEl.value === '' ? null : parseInt(floorEl.value)
if (floor !== null && (isNaN(floor) || floor < 3 || floor > 18)) {
```